docs/CodingStyleGuidelines.md

```diff
@@ -736,28 +736,28 @@ The above method for passing virtual interfaces to the verification environment
 
 4.4.3. Components only get handles to virtual interfaces from the UVM resource database.
 
-4.4.4. The component defines a string configuration field that is used to look up the virtual interface (`intf_name`, by custom).
+4.4.4. The component defines a string configuration field that is used to look up the virtual interface (`if_name`, by custom).
 
 4.4.5. The vkit defines a scope name under which all virtual interfaces for all instances of the verification kit are stored.
 
-4.4.6. *Recommended*: Use the macro `` `cmn_get_intf `` to pull virtual interface handles from the resource database.
+4.4.6. *Recommended*: Use the macro `` `cmn_get_if `` to pull virtual interface handles from the resource database.
 
 4.4.7. *Recommended*: Components should use the `` `uvm_component_utils `` macros to declare configurable variables which perform database lookups during the build phase. Such variables should have the `UVM_COMPONENT` flag attached.
 
-Extending the example from above, the `foo_pkg::env_c` class has a configuration field called "`intf_name`". Wherever a FOO class needs to get the virtual interface to a `foo_intf`, it knows to look in the UVM resource database under the scope `foo_pkg::intf`, and the name specified by the `intf_name` field. If the interface was not registered or was registered with a different name, a fatal error prints out and reports that `<UNSPECIFIED>` is not found in the database.
+Extending the example from above, the `foo_pkg::env_c` class has a configuration field called "`if_name`". Wherever a FOO class needs to get the virtual interface to a `foo_if`, it knows to look in the UVM resource database under the scope `foo_pkg::if`, and the name specified by the `if_name` field. If the interface was not registered or was registered with a different name, a fatal error prints out and reports that `<UNSPECIFIED>` is not found in the database.
 
 ```v
 class env_c;
    `uvm_component_utils_begin(env_c)
-      `uvm_field_string(intf_name, UVM_COMPONENT)
+      `uvm_field_string(if_name, UVM_COMPONENT)
       //...
 
-   string intf_name = "<UNSPECIFIED>";
-   virtual foo_intf foo_vi;
+   string if_name = "<UNSPECIFIED>";
+   virtual foo_if foo_vi; //TODO: vi= viritual interface, should this be if_vi?
```

Comment:
If we decide to use the _if_i or _if as a standard for interface instances, we should probably also change _vi to _if_vi or _vif for the virtual interfaces as well, or what do you think? 

(I see I have misspelled virtual in the comment... When writing viritual, I mean to "say" virtual)

Reply:
We do use `..._vif`.